server/events/models/models.go

```diff
@@ -442,6 +451,22 @@ type PlanSuccess struct {
 	HasDiverged bool
 }
 
+// PolicyCheckSuccess is the result of a successful policy check run.
+type PolicyCheckSuccess struct {
+	// PolicyCheckOutput is the output from policy check binary(conftest|opa)
+	PolicyCheckOutput string
+	// LockURL is the full URL to the lock held by this policy check.
+	LockURL string
+	// RePlanCmd is the command that users should run to re-plan this project.
+	RePlanCmd string
+	// ApplyCmd is the command that users should run to apply this plan.
+	ApplyCmd string
```

Comment:
If we have policy checks enabled, we should prevent showing the apply command in the plan output. We can do this as a follow up but just calling it out.